pkgs/development/python-modules/manage-django-project/default.nix

```diff
@@ -0,0 +1,121 @@
+{
+  python,
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-scm,
+  django,
+  django-tools,
+  cli-base-utilities,
+  manageprojects,
+  bx-py-utils,
+  django-rich,
+  cmd2,
+  pytestCheckHook,
+  pytest-django,
+  cmd2-ext-test,
+  typeguard,
+  pip,
+  pip-tools,
+  coverage,
+  tox,
+  darker,
+  flake8,
+  pip-audit,
+}:
+
+buildPythonPackage rec {
+  pname = "manage-django-project";
+  version = "0.8.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jedie";
+    repo = "manage_django_project";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-HkosxxFsX5juqKqwbF+2jTCsiKa1Y+zADFjxdlRBBVk=";
+  };
+
+  postPatch = ''
+    substituteInPlace manage_django_project/tests/test_command_coverage.py \
+    manage_django_project/tests/test_command_tox.py \
+    manage_django_project/tests/test_command_update_test_snapshot_files.py \
+      --replace-fail '.../bin/coverage' '${coverage}/bin/coverage'
```

Comment:
Could be better to patch https://github.com/jedie/manage_django_project/blob/a07806a0304c0106f8378fad69b91de5c52f0b16/manage_django_project/tests/command_test_utils.py#L13 instead to strip the nix store path, so we don't have to patch all the separate tests.